macros/storage/important-bucket-policy.mdx

```diff
@@ -0,0 +1,9 @@
+---
+macro: important-bucket-policy
+---
+
+<Message type="important">
+    - Each bucket can have only one bucket policy.
+    - Pushing a new bucket policy to a bucket overwrites any existing bucket policy.
```

Comment:
Since there is only one bucket policy, we could say "If there is already a bucket policy on the bucket, pushing a new bucket policy overwrites it".
But i'm fine with current wording

Reply:
I'd like to keep the two separate explicit sentences because some users may look for one of these info

Reply:
I meant "replace the second point by my proposal" :p but as said current proposal is fine